exchanges/okex/okex_test.go

```diff
@@ -1551,26 +1551,6 @@ func TestGetETTOrderDetails(t *testing.T) {
 	testStandardErrorHandling(t, err)
 }
 
-// TestGetETTConstituents API endpoint test
```

Comment:
Can t.Skip these, if they don't bring them back online then we can remove the functions and associated tests.

Reply:
Far out that wasn't thought through 😄 "didn't work, better remove the tests!" Will add back. Can remove ETT if it doesn't return in another PR later 😄